docs/API.md

```diff
@@ -1924,7 +1924,7 @@ curl \
     * `uid` : hex
 * Params:
     * `sharedSecret` : hex10
-    * `epoch` : epoch    
```

Comment:
Webstorm was picking up that there were some lines that had trailing spaces. It is unrelated to PR but since the change is small, I figured I could leave it in so doc is consistent.